packages/graph/hash_graph/lib/graph/src/api/rest/entity.rs

```diff
@@ -215,29 +220,13 @@ async fn archive_entity<P: StorePool + Send>(
     pool: Extension<Arc<P>>,
     body: Json<ArchiveEntityRequest>,
 ) -> Result<(), StatusCode> {
-    let Json(ArchiveEntityRequest {
-        entity_id,
-        actor_id,
-    }) = body;
-
-    let mut store = pool.acquire().await.map_err(|report| {
-        tracing::error!(error=?report, "Could not acquire store");
-        StatusCode::INTERNAL_SERVER_ERROR
-    })?;
-
-    store
-        .archive_entity(entity_id, actor_id)
-        .await
-        .map_err(|report| {
-            if report.contains::<QueryError>() {
-                return StatusCode::NOT_FOUND;
-            }
-
-            tracing::error!(error=?report, "Could not archive entity");
-
-            // Insertion/update errors are considered internal server errors.
-            StatusCode::INTERNAL_SERVER_ERROR
-        })
+    tracing::warn!(
+        "Use of deprecated endpoint `/entities/archive`, use `/entities/update` instead"
```

Comment:
There should be a OpenAPI flag for deprecation. No idea how to set it in utoipa though - and not worth doing if this function is to be removed entirely within the dev-branch

Reply:
I don't know if this function is going to be replaced in the `dev` branch or shortly after. It's going to be replaced in the linked task, which I hopefully get to this week.

Reply:
I think it will have to be replaced as part of the dev branch if we do not reimplement it.